backend/addcorpus/es_mappings.py

```diff
@@ -100,5 +100,5 @@ def geo_mapping():
     return {'type': 'geo_point'}
 
 
-def annotated_text_mapping():
-    return {'type': 'annotated_text'}
+def ner_mapping():
+    return {'type': 'text', 'index': False}
```

Comment:
What was the issue with the 'annotated_text' type?

Reply:
See https://github.com/CentreForDigitalHumanities/I-analyzer/issues/1724 

TLDR: it wasn't necessary because we already have the unannotated text for full-text search, and keyword fields for finding entities.